h2o-r/h2o-package/R/gbm.R

```diff
@@ -111,7 +111,7 @@
 #' # Run regression GBM on australia data
 #' australia_path <- system.file("extdata", "australia.csv", package = "h2o")
 #' australia <- h2o.uploadFile(path = australia_path)
-#' independent <- c("premax", "salmax","minairtemp", "maxairtemp", "maxsst",
```

Comment:
This is the wrong file to update. The changes to this and other algos have to be made in:
/h2o-3/h2o-bindings/bin/custom/R